samples/ionic/src/app/hivejs/payment/logger.ts

```diff
@@ -0,0 +1,77 @@
+export enum LogLevel {
+    NONE,
+    ERROR,
+    WARN,
+    INFO,
+    DEBUG,
+}
+
+export class Logger {
```

Comment:
There's already a logger in Hive JS. Why creating a new one? I'm in the process of moving all shareable code to a single utility library, and the Logger will be part of it. It will add a logging standard accross all apps.

Reply:
some classes from fold payment come from the Feeds app, will optimize this.